docs/github-action.md

```diff
@@ -11,7 +11,7 @@ nav_order: 6
 
 OSV-Scanner is offered as a GitHub Action. We currently have two different GitHub Actions:
 
-1. An action that performs a vulnerability scan on a [regular schedule](./github-action.md#scheduled-scans).
+1. An action that performs a single vulnerability scan, which can be configured to scan on a [regular schedule](./github-action.md#scheduled-scans), or used as a check [on releases](./github-action.md#scan-on-release) to prevent releasing with known vulnerabilities in dependencies.
```

Comment:
Hey @another-rex . 

I tried to add commits to the PR but I don't think I have the right permissions, so I am going to comment instead. 

Please either:
1. Make these changes before merging
2. Update permissions so I can add commits. I tried to follow [these instructions] (https://tighten.com/insights/adding-commits-to-a-pull-request/) and got an error message that said "fatal: Could not read from remote repository.Please make sure you have the correct access rights and the repository exists." 
3. Merge it anyway and I can do another PR with my updates. Nothing is wrong, just not optimal so it would be ok to merge.

Reply:
Thanks for the comments! Added permissions to allow you to push, do you want me to make the changes you proposed, or would you like to push the changes on your side, and I'll just review them?